### ISSUE TITLE
Update numpy

### ISSUE THREAD

**Issue by CiaranWelsh** [sys-bio/roadrunner#758]
As per the comment in #757, our pinned version of numpy needs updating.

**Comment by luciansmith** [comment]
Done!  We actually had to fix this in tellurium, not roadrunner, though rr was also updated.

**Comment by luciansmith** [comment]
That... broke everything.  @CiaranWelsh , can you look into this?  Maybe Mattias was just wrong about it requiring that version of numpy?

**Comment by matthiaskoenig** [comment]
For me the roadrunner many linux py37 wheel worked with 1.20.2 but not
1.19.3. Perhaps the wheels are build with different versions of numpy.
This was reproducible. I.e changing versions broke or fixed roadrunner.

**Comment by CiaranWelsh** [comment]
No, this wasn't the cause of the failed build. The problem is that I updated libsbml and poco in the dependencies but haven't merged the corresponding fixes to the main roadrunner repo yet. (The roadrunner azure simply grabs the newest sucessful build from the dependencies azure). 

In short, I've done a lot, but this is taking a little longer than I anticipated. I gave the tests a bit of a makeover (because of some instability) and I'm just trying to fix the logger which is broken since the poco update. The poco stuff is a little confusing but I hope to get it finished later tonight or tomorrow.
